gbpservice/neutron/nsf/core/lb.py

```diff
@@ -0,0 +1,46 @@
+import os
+import sys
+
+
+class RoundRobin(object):
+
+    def __init__(self, workers):
+        self._workers = workers
+        self._rridx = 0
+        self._rrsize = len(self._workers)
+
+    def _rr(self):
+        item = self._workers[self._rridx]
+        self._rridx = (self._rridx + 1) % (self._rrsize)
+        return item
+
+    def get(self, rsrcid):
+        return self._rr()
+
+    def rem(self, rsrcid):
+        pass
```

Comment:
Actually no one.. before writing the logic, had put template for the class with "put,get,rem" etc methods.. will remove it.